src/shared/scrapers/UKR/index.js

```diff
@@ -0,0 +1,99 @@
+import * as fetch from '../../lib/fetch/index.js';
+import datetime from '../../lib/datetime/index.js';
+import maintainers from '../../lib/maintainers.js';
+import * as geography from '../../lib/geography/index.js';
+import * as parse from '../../lib/parse.js';
+
+const iso2ref = {
+  'Autonomous Republic of Crimea': 'iso2:UA-43',
+  'Cherkasy region': 'iso2:UA-71',
+  'Chernihiv region': 'iso2:UA-74',
+  'Chernivtsi region': 'iso2:UA-77',
+  'Dnipropetrovsk region': 'iso2:UA-12',
+  'Donetsk region': 'iso2:UA-14',
+  'Ivano-Frankivsk region': 'iso2:UA-26',
+  'Kharkiv region': 'iso2:UA-63',
+  'Kherson region': 'iso2:UA-65',
+
+  'Khmelnytskyi region': 'iso2:UA-68',
+  'Khmelnytsky region': 'iso2:UA-68',
+
+  'Kirovohrad region': 'iso2:UA-35',
+  'Kirovograd region': 'iso2:UA-35',
+
+  Kyiv: 'iso2:UA-30',
+  'Kyiv city': 'iso2:UA-30',
+
+  'Kyiv region': 'iso2:UA-32',
+  'Kiev region': 'iso2:UA-32',
+
+  'Luhansk region': 'iso2:UA-09',
+  'Lugansk region': 'iso2:UA-09',
+
+  'Lviv region': 'iso2:UA-46',
+  'Mykolaiv region': 'iso2:UA-48',
+
+  'Odesa region': 'iso2:UA-51',
+  'Odessa region': 'iso2:UA-51',
+
+  'Poltava region': 'iso2:UA-53',
+  'Rivne region': 'iso2:UA-56',
+  Sevastopol: 'iso2:UA-40',
```

Comment:
Also, isn't it "Sevastopol region"?

Reply:
looking into this. I think some regions aren't included. I thought there was a mismatch between the list you gave me and what the JSON had.

Reply:
`iso2:UA:40` is a city https://en.wikipedia.org/wiki/ISO_3166-2:UA